lifecycle-operator/apis/options/v1alpha1/keptnconfig_types.go

```diff
@@ -41,6 +41,7 @@ type KeptnConfigSpec struct {
 	// CloudEventsEndpoint can be used to set the endpoint where Cloud Events should be posted by the lifecycle operator
 	// +optional
 	CloudEventsEndpoint string `json:"cloudEventsEndpoint,omitempty"`
+	BlockDeployment     bool   `json:"blockDeployment, omitempty"`
```

Comment:
let's introduce a default value here set to true here

Reply:
Default should be `false` shouldn't it? If not specified, don't block?